src/main/java/com/emc/object/util/RestUtil.java

```diff
@@ -206,6 +211,23 @@ public static Date headerParse(String dateString) {
         }
     }
 
+    public static Date headerParse(String dateString, String headerKey) {
```

Comment:
I don't see a reason to have the `headerKey` param - we have `headerParse()` method for non-AMZ parsing - this method can be called `amzHeaderParse()` for AMZ-specific header parsing

Reply:
Changed.